style.css

```diff
@@ -412,3 +424,199 @@ button {
   padding: 12px;
   color: #396df2;
 }
+
+.navDesk {
+  display: none;
+}
+
+@media screen and (max-width: 480px) {
+  .card5,
+  .card6,
+  .card7 {
+    display: none;
+  }
+}
+
+@media screen and (min-width: 992px) {
+  .menu-icon {
+    display: none;
+  }
+
+  .first-page {
+    background-image: none;
+  }
+
+  .logo {
+    width: 600px;
+    justify-content: start;
+    margin-top: 150px;
+    margin-left: 140px;
+  }
+
+  .header-container {
+    display: inline;
+  }
+
+  .navDesk {
+    display: flex;
+    width: 50%;
+    margin-left: 50%;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .menuDesk {
+    display: inline;
+  }
+
+  .menuDesk a {
+    font-family: Poppins, sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+    list-style: none;
+    margin-right: 16px;
+    text-decoration: none;
+  }
+
+  .headline {
+    background-image: url('src/header-shapes-mobile.svg');
+    background-size: 90%;
+    background-position: 0% 15%;
+    align-items: flex-start;
+    justify-content: center;
+    padding-left: 140px;
+    padding-bottom: 150px;
+    border-bottom-left-radius: 165px;
+  }
+
+  .title {
+    width: 600px;
+    line-height: 60px;
+    justify-content: start;
+  }
+
+  .paragraph {
+    width: 85%;
+    line-height: 28px;
+    font-size: 20px;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .mobileimg {
+    display: none;
+  }
+
+  .card2,
+  .card3,
+  .card4 {
+    display: none;
+  }
+
+  .project {
+    display: grid;
+    column-gap: 30px;
+  }
+
+  .image {
+    width: 544px;
+    height: 448px;
+    grid-area: 1/1/3/2;
+  }
+
+  .description {
+    grid-area: 1/2/3/3;
+  }
+
+  .imageReverse {
+    width: 544px;
+    height: 448px;
+    grid-area: 1/2/3/3;
+  }
+
+  .descriptionReverse {
+    grid-area: 1/1/3/2;
+  }
+
+  .about-section {
+    display: grid;
+  }
```

Comment:
- [ ] Kindly here you have to make the first part with width 40% and the second 60%, to be like the Figma design.

| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/35535130/155708179-f20263e9-e174-4be7-ab27-e72fbc1ff674.png) | ![image](https://user-images.githubusercontent.com/35535130/155708236-f78eb417-f81e-4f24-a0ff-9af733cb6b6d.png) |